### DISCUSSION TITLE
How can dummy daddy use PageFind in nuxt?

### DISCUSSION THREAD

**Discussion by LutherAdvocate** [Pagefind/pagefind#227] [Q&A]
Hey! 
The problemm is how to use PageFind in a `nuxt`-development, who deploys automatic on `Vercel` after I pushed it to github.
... 
I have worked with this problem the last two days. The documentation have no examples (for dummies like me) to learn from.

After I find [an overview of 63 repos who depend on pagefind](https://github.com/CloudCannon/pagefind/network/dependents), and study the `package.json`-file in some of them, I finally figured out that the key is how the script about pagefind is setup.

The script-setup in my `package.json` is now like this:
```
"scripts": {
    "dev": "nuxi dev",
    "generate": "nuxt generate && npx pagefind",
    /* more scripts */
  }
```
And then I created `pagefind.yml`-file with this content:
```
source: .output/public
bundle_dir: ../../public/_pagefind
```
**The process to make it work was like this:**
`yarn generate` creates first the html-files and then it generate the _bundle_-dir in the _public_-directory on the root level. 

If no errors on _yarn generate_, `push` the development from the `local`-machine to `GitHub`, and from there to `Vercel`.

[Check it out here](https://kirkepostille.vercel.app/).
Sorry about my English! Thats why I had to choose law studies in Norwegian:)_

**Comment by bglw** [comment]
Hi @lovkyndig — sorry the delay on replying here! That site is looking great, I'm glad you were able to resolve the issues by looking at the reference repositories.

Feel free to drop any other questions you have here and I'll try to be more prompt in replying!

**Reply by LutherAdvocate** [reply]
Thanks!
Now I'm working with the problem to continue  the search in an selected page, after jusing PageFind on all the catalogs.
This is improtant, because every page takes from 30 to 45 minutes to read.
I'm using `window.find()`, but if somone else know if there is possibly to us _PageFind_ on one page, and how to do it, it would be very nice to know:)
